test/integration/targets/loop_control/runme.sh

```diff
@@ -9,3 +9,14 @@ MATCH='foo_label
 bar_label'
 [ "$(ansible-playbook label.yml "$@" |grep 'item='|sed -e 's/^.*(item=looped_var \(.*\)).*$/\1/')" == "${MATCH}" ]
 
+# user output has:
+#ok: [localhost] => {
+#    "msg": "1 0 3 2 True False 3 first,second,third"
+#ok: [localhost] => {
+#    "msg": "2 1 2 1 False False 3 first,second,third"
+#ok: [localhost] => {
+#    "msg": "3 2 1 0 False True 3 first,second,third"
+MATCH='"1 0 3 2 True False 3 first,second,third"
+"2 1 2 1 False False 3 first,second,third"
+"3 2 1 0 False True 3 first,second,third"'
+[ "$(ansible-playbook extended.yml "$@" |grep '"msg":'|sed -e 's/^.*"msg": \(.*\)$/\1/')" == "${MATCH}" ]
```

Comment:
These types of tests are always hard to diagnose failures with.  I'd rather this exist in the playbook itself.  The `ansible_loop` tests, use assert on all of the values, so I'm not sure we need to assert that each are still the same.